### DISCUSSION TITLE
TypeError: add_vehicle_type() got an unexpected keyword argument 'unit_distance_cost'

### DISCUSSION THREAD

**Discussion by wzhuwz** [PyVRP/PyVRP#602] [Q&A] [answered]
I am using the pyvrp to solve a prize collecting routing problem. The objective is to collect more prize and do not consider the travelling distance. Hence, We want to set the "unit_distance_cost" to zero by adding parameters "unit_distance_cost=1" to add_vehicle_type(). However, "TypeError: add_vehicle_type() got an unexpected keyword argument 'unit_distance_cost'" occurs. Is there ways to set this? thanks!

**Comment by N-Wouda** [comment, marked as answer]
Hi @wzhuwz!

>However, "TypeError: add_vehicle_type() got an unexpected keyword argument 'unit_distance_cost'" occurs. Is there ways to set this?

This feature is not yet released on PyPI. If you want to use this now, I suggest you install directly from GitHub by either following the instructions [here](https://pyvrp.org/setup/installation.html#installing-from-source), or, if compiling turns out to be an issue, installing the appropriate precompiled wheel from our CD runs [here](https://github.com/PyVRP/PyVRP/actions/workflows/CD.yml). Currently, the latest run is [this one](https://github.com/PyVRP/PyVRP/actions/runs/9345309952): download the artefact belonging to your OS, unzip it, and install the wheel appropriate for your Python version.

**Reply by wzhuwz** [reply]
Thanks a lot!
